spec/apis/external_service_connectivity_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+feature 'app is connected to multiple third party APIs', %{
+  As site manager
+  I want to quickly check that essential API syntax has not changed
+  So that I can troubleshoot more efficiently
+}, dashboard: true do
+
+  scenario "app connects to google geocoding API and returns lat and lng" do
+    query = HTTParty.get("https://maps.googleapis.com/maps/api/geocode/json?address=1600+Amphitheatre+Parkway,+Mountain+View,+CA&key=#{ENV["GOOGLE_GEOCODING"]}")
+    expect(query.first[1].first["geometry"]["location"]["lat"].round(2)).to eq(37.42)
+    expect(query.first[1].first["geometry"]["location"]["lng"].round(2)).to eq(-122.08)
+  end
+
+  scenario "app connects to yahoo WOEID API and returns lat and lng" do
+    query = HTTParty.get("http://where.yahooapis.com/v1/place/2367105?format=json&appid=#{ENV["YAHOO"]}")
```

Comment:
Line is too long. [105/80]<br>Prefer single-quoted strings inside interpolations.